Assets/Dependencies/SmashBrew/Character/States/MovementState.cs

```diff
@@ -168,16 +168,18 @@ struct MovementInfo {
 
         void SnapToLedge() {
             IsFastFalling = false;
+            var ledge = CurrentLedge.GetComponentInParent<StageLedge>();
```

Comment:
GetComponentInParent is an expensive call to make every frame, since this is also called every frame to keep the character snapped to the ledge.

GameObject already has a `transform` property, and can be used instead of fetching the StageLedge.transform.